src/parser/shared/modules/spells/bfa/azeritetraits/EphemeralRecovery.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Analyzer from 'parser/core/Analyzer';
+import SPELLS from 'common/SPELLS/index';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'interface/others/TraitStatisticBox';
+import ItemManaGained from 'interface/others/ItemManaGained';
+
+/**
+ Your heals have a chance to apply Bracing Chill. Healing a target with Bracing Chill will heal for an additional 1155 and move Bracing Chill to a nearby ally (up to 6 times).
```

Comment:
looks like a copypaste error with the ephemeral recovery comment

Reply:
Fixed